docs/documentation/api-specification/extension-api.md

```diff
@@ -26,7 +26,46 @@ Checklist for requesting approval for your extensions.
 **Category**: Generic  
 ---
 ## Description
+The Extension API provides information regarding the XtendM3 extension that is currently in use. This can be used as a validation in the development of the extension code as there can be other concurrent XtendM3 extensions activated.
```

Comment:
```suggestion
The Extension API provides information regarding the XtendM3 extension that is currently in use. This can be used as to retrieve metadata about the current extension and show to the user or use for logging purposes.
```